src/libraries/System.Private.Xml/src/System/Xml/Xsl/Xslt/XslAstAnalyzer.cs

```diff
@@ -1134,7 +1134,9 @@ public XslFlags Function(string prefix, string name, IList<XslFlags> args)
                     {
                         // Unknown function. Can be script function or extension function
                         funcFlags = XslFlags.AnyType;
+#pragma warning disable SYSLIB0062 // XsltSettings.EnableScript is obsolete
                         if (_compiler.Settings.EnableScript && ns != null)
+#pragma warning restore SYSLIB0062
                         {
                             XmlExtensionFunction? scrFunc = _compiler.Scripts.ResolveFunction(name, ns, args.Count, default(NullErrorHelper));
```

Comment:
`_compiler.Scripts` is internal, so not relevant to the user. But is it worth obsoleting it as well for our own internal awareness?